test/UsersTest.cs

```diff
@@ -33,16 +33,15 @@ public void TestListUser()
 
             // Verify parameters of GetListVariables() passed as expected toward conjur server
             ClearMocker();
-            Client.ActingAs = "user:role";
             Mocker.Mock(new Uri($"{userListUrl}?offset=0&limit=1000&search=user_0&acting_as=user:role"), GenerateUsersInfo(0, 1000));
             Mocker.Mock(new Uri($"{userListUrl}?offset=1000&limit=1000&search=user_0&acting_as=user:role"), GenerateUsersInfo(1000, 1872));
             Mocker.Mock(new Uri($"{userListUrl}?offset=1872&limit=1000&search=user_0&acting_as=user:role"), "[]");
-            users =(Client.ListUsers("user_0")).GetEnumerator();
+            users =(Client.ListUsers("user_0", "user:role")).GetEnumerator();
```

Comment:
This doesn't comply with the example: "MyCompanyName:group:security_admin".

Reply:
Here we only check that the parameter counts.
We don't work with real conjur, I think it is not so important here.

Reply:
I agree, that's why I approved. If we touch this in the future we can fix it.